ccf_transaction_processor/scripts/configure_ccf_network.py

```diff
@@ -56,11 +71,42 @@ def open_network_script(client, options, config) :
         sys.exit(-1)
 
     if r.status != http.HTTPStatus.OK.value:
+        LOG.info(r)
         LOG.error('failed to open network: {}'.format(r.status))
         sys.exit(-1)
```

Comment:
if we really want r to debug, then just throw it into the error log.

Reply:
thank you, this one should not have been there, I was most likely debugging something but missed to remove, I will remove this.